### DISCUSSION TITLE
Can I configure a redirect for an existing blog post?

### DISCUSSION THREAD

**Discussion by tpcarman** [squidfunk/mkdocs-material#6215] [Q&A]
I have made a slight change to the title of an existing blog post. Is it possible to configure a redirection to the updated post using the previously shared link?

I have attempted to use mkdocs-redirects plugin, however I am not having any success.

**Comment by alexvoss** [comment]
Hi, I am currently working on examples for the blog plugin and this seems like an interesting case for me to pick up. Can you do me the favour and describe in a bit more detail what you have tried and how it did not work? A [minimal repdoduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) would be ideal. I will have some time in a few hours to look at this and would like to make sure the example gives you the information you need.

**Comment by tpcarman** [comment]
Hi @alexvoss,
Unfortunately I don't have time right now to setup a minimal reproduction however I am happy to provide more detail about my issue. I can likely provide a reproduction in a few days when I have more time.
The issue I have is that a have created a page for a blog post e.g. `my-original-blog-post.md` in my blog_dir, and now I wish to rename this page e.g. `my-renamed-blog-post.md` whilst maintaining access via the original blog post link `https://www.myblog.com/blog/2023/08/25/my-original-blog-post/`

Thanks for the assistance.

**Reply by alexvoss** [reply]
Hi, I just stumbled across you post again and wanted to check if you have solved the problem? Have you tried the [redirect plugin](https://github.com/mkdocs/mkdocs-redirects)?
